SIPS/sip-62.md

```diff
@@ -0,0 +1,60 @@
+
+## Motivation
+The creation of a single reference price for futures markets that incorporates information from the most liquid futures contracts enables assets like WTI and other commodities to be traded as synthetic assets within Synthetix. There is significant demand for these assets particularly given the liquidity of the underlying markets and the difficulty of access for the average trader. 
+ 
+## Specification
+There are many markets where price discovery occurs solely or predominantly in the futures markets, these futures markets are made up of individual contracts with varying expiry dates. This presents a problem when attempting to create a single reference price. The solution proposed is to employ a dynamic weighting scheme (in continuous time) of the near two contract months, with emphasis initially given to the near contract.  As expiry approaches, however, weight is progressively shifted out of the near contract in favor of the 2nd month.  Moreover, upon reaching 5 days (configurable via SCCP) prior the last trade time (2:30PM EST on the exchange stipulated Last Trade day), zero weighting in the near contract is achieved, with the weight instead being allocated between the 2nd and 3rd contract months.  Once the front month expires, the next nearest two live contracts become the 1st and 2nd months, and the dynamic weighting process repeats. This weighting methodology is intentionally relatively simple and linear to enable easier reasoning about the likely behaviour of the resulting reference price from various futures contract situations such as contango and backwardation, and specifically super-contango observed in early 2020 due to extreme volatility in the front month expiring WTI contracts resulting in negative futures prices.
+
+The formula for the reference price is:
+
+<img width="441" alt="Screen Shot 2020-05-27 at 8 41 07 pm" src="https://user-images.githubusercontent.com/1691389/83010063-097ee280-a05b-11ea-9a84-714e0deefa5b.png">
+
+\\[ 
+price = \begin{cases}
+\frac{d_{1} \ - \ X}{d_{1} \ - \ d_{0}} \cdot P_1 \ + \ \frac{d_{0} \ + \ X}{d_{1} \ - \ d_{0}} \cdot P_2 &   \ \mbox{if } \ \ \  X \leq d_1 \\
+\\
+\frac{d_{2} \ - \ X}{d_{2} \ - \ d_{1}} \cdot P_2 \ + \ \frac{X \ - \ d_{1}}{d_{2} \ - \ d_{1}} \cdot P_3 & \ \mbox{if } \ \ \ 0 \lt d_1 \lt X \\
+
+\end{cases}
+\\]
+
+With the inputs below:
+X: # of days prior to expiration to achieve zero weight in in the expiring contract (default of 5 days)
+DTE0: Days since the prior month contract expired
+DTE1: Days remaining for the current front month contact
+DTE2: Days remaining for the current 2nd month contract
+P0: Orderbook mid-price of the current front month contract
+P1: Orderbook mid-price of the current 2nd month contract
+p2: Orderbook mid-price of the current 3rd month contract
```

Comment:
Because these represent formula variables, I think casing is important to keep them consistent - so all the `p` and `d` entries should be lowercase to match the formula spec

Reply:
will do.